products/BellHybrid/apps/application-bell-main/widgets/BellBattery.cpp

```diff
@@ -8,7 +8,8 @@
 namespace
 {
     constexpr auto entries =
-        std::array<battery_utils::BatteryLevelEntry, 6>{{{{5, 10}, {1, 5}, "bell_battery_empty"},
+        std::array<battery_utils::BatteryLevelEntry, 7>{{{{0, 5}, {1, 1}, "bell_status_battery_lvl0"},
+                                                         {{5, 10}, {1, 5}, "bell_battery_empty"},
```

Comment:
I believe `bell_status_battery_lvl0` `bell_battery_empty` are of different sizes and probably shouldn't be combined.

Reply:
Yes, it is an error (copied from home screen). Fixing.